### PR TITLE
added zva-wmk again after review

### PR DESCRIPTION
**Pull request by Demel75**

### CODE REVIEW

File: custom_components/waste_collection_schedule/waste_collection_schedule/source/zva_wmk_de.py [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    def fetch(self):
+        city = self._city.replace('ß', 'ẞ').upper()
+        city = city.replace(" - ", "_")
+        city = city.replace(" ", "+")
+        city = city.replace("ẞ", "ß")
+        street = self._street
+        street = street.replace(" ","+")
+        today = datetime.date.today()
+        year = today.year
+        year = 2023
```
Comment (mampfes) [inline custom_components/waste_collection_schedule/waste_collection_schedule/source/zva_wmk_de.py:33]: Hm, what's about 2024?

File: custom_components/waste_collection_schedule/waste_collection_schedule/source/zva_wmk_de.py [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+           yearstr = ""
+           street = self._street.upper()
+        else:
+           yearstr = ("-" + str(year))
+        payload = {"city": city, "street": street}
+        urlzva = "https://www.zva-wmk.de/termine/schnellsuche"+yearstr+"&type=all&link=ical&timestart=6&fullday=1&timeend=17&reminder=1440&display=0"
+
+        r = requests.get(urlzva, params=payload)
+        
+        r.encoding = r.apparent_encoding
```
Comment (mampfes) [inline custom_components/waste_collection_schedule/waste_collection_schedule/source/zva_wmk_de.py:44]: Don't use apparent_encoding, but set correct encoding (probably utf-8) directly.

File: custom_components/waste_collection_schedule/waste_collection_schedule/source/zva_wmk_de.py [hunk 3]
```diff
@@ -0,0 +1,50 @@
+import requests
+import datetime
+from waste_collection_schedule import Collection  # type: ignore[attr-defined]
+from waste_collection_schedule.service.ICS import ICS
+
+import urllib
```
Comment (mampfes) [inline custom_components/waste_collection_schedule/waste_collection_schedule/source/zva_wmk_de.py:6]: obsolete?

**Review by mampfes — CHANGES_REQUESTED** [review]
Please add a link to the new source documentation to README.md and info.md

File: custom_components/waste_collection_schedule/waste_collection_schedule/source/zva_wmk_de.py (hunk 1 above)
Comment (Demel75) [reply custom_components/waste_collection_schedule/waste_collection_schedule/source/zva_wmk_de.py:33]: dam, was for testing. thanx.

File: custom_components/waste_collection_schedule/waste_collection_schedule/source/zva_wmk_de.py (hunk 3 above)
Comment (Demel75) [reply custom_components/waste_collection_schedule/waste_collection_schedule/source/zva_wmk_de.py:6]: don't know - used erlangen_hoechstadt_de.py  as inspiration

File: custom_components/waste_collection_schedule/waste_collection_schedule/source/zva_wmk_de.py (hunk 2 above)
Comment (Demel75) [reply custom_components/waste_collection_schedule/waste_collection_schedule/source/zva_wmk_de.py:44]: it was also used in erlangen_hoechstadt_de.py that was my inspiration

**Comment by Demel75** [comment]
Hi, I tired my best with this github stuff. Anything missing?

File: custom_components/waste_collection_schedule/waste_collection_schedule/source/zva_wmk_de.py (hunk 3 above)
Comment (mampfes) [reply custom_components/waste_collection_schedule/waste_collection_schedule/source/zva_wmk_de.py:6]: I didn't see that it is used, therefore should be obsolete. Please remove it and test it everythin is still working.

File: custom_components/waste_collection_schedule/waste_collection_schedule/source/zva_wmk_de.py (hunk 2 above)
Comment (mampfes) [reply custom_components/waste_collection_schedule/waste_collection_schedule/source/zva_wmk_de.py:43]: Ok, then it is probably not needed. Please remove it and test if everything is still working (especially German Umlaute).

**Review by mampfes — CHANGES_REQUESTED** [review]
Please add link also to README.md. This is still missing.

**Comment by Demel75** [comment]
hopefully thats all for now. Thanks for your patience.

**Comment by mampfes** [comment]
Thanks for your contribution.

**Pull request merged**